### PR TITLE
Announce PHP 5.4+ Minimum Requirement and APC Extension Not Supported

### PR DESCRIPTION
**Pull request by raamdev**

### CODE REVIEW

**Comment by raamdev** [comment]
@jaswsinc Review, please.

**Comment by jaswsinc** [comment]
Reviewed. Looks great to me!

File: plugin.php [hunk 1]
```diff
@@ -5,33 +5,60 @@
...
+            ${__FILE__}['php54_warning'] .= '<p style="margin-top:0;">'.sprintf(__('As of December 1st, 2015 ZenCache will require PHP 5.4 or higher.</strong> The minimum PHP version required to run ZenCache is being increased from PHP 5.3.2+ to PHP 5.4+. It appears that your server is currently running PHP v%1$s. You will need to upgrade to PHP 5.4 or higher to run the next version of ZenCache.', 'zencache'), esc_html(PHP_VERSION)).'</p>';
+            ${__FILE__}['php54_warning'] .= '<p style="margin-top:0;">'.__('To learn more about this upcoming change, please see the announcement: <a href="http://zencache.com/r/new-minimum-php-version-php-5-4/" target="_blank">New Minimum PHP Version: PHP 5.4</a>', 'zencache').'</p>';
+            if ($_apc_enabled) {
+                ${__FILE__}['php54_warning'] .= '<p style="margin-top:0;">'.__('Additionally, it appears that your server is also running the outdated PHP APC extension. <strong>The PHP APC extension will also no longer be supported as of December 1st, 2015.</strong> Please see: <a href="">PHP APC Extension No Longer Supported</a>', 'zencache').'</p>';
```
Comment (raamdev) [inline plugin.php]: @jaswsinc Yikes, I'm glad that I decided to rewrite this message. I just noticed that we both missed the missing `href` link on this line! :-)

Comment (jaswsinc) [inline plugin.php]: Aha! I see what you mean now; i.e., `<a href="">`. Yep, I missed that before. Sorry.

**Pull request merged**